lib/smart-survey-api.js

```diff
@@ -2,20 +2,21 @@
 
 const SmartSurveyClient = require('smartsurvey-client');
 
+const createClient = (token, tokenSecret) => {
+    return new SmartSurveyClient({ apiToken: token, apiTokenSecret: tokenSecret});
+};
```

Comment:
I _think_ you could lose the braces and have this as a one-liner:
```js
const createClient = (token, tokenSecret) => new SmartSurveyClient({ apiToken: token, apiTokenSecret: tokenSecret});
```
Of course I could be getting mixed up with Scala again.